dds/DdsDcpsCore.idl

```diff
@@ -24,546 +22,563 @@
 #  define BUILT_IN_TOPIC_KEY
 #endif
 
-module DDS {
 // Make the IDL compiler produce include of zero-copy read info
 // sequence template.
 #pragma DCPS_SUPPORT_ZERO_COPY_READ
 
-    typedef sequence<string> StringSeq;
-    typedef HANDLE_TYPE_NATIVE InstanceHandle_t;
+module DDS {
+  // Typedefs that fulfil XTypes 1.3 Table 43
```

Comment:
"fulfill"
Or just "XTypes 1.3 Table 43"

Reply:
Fulfil is also a correct spelling